tsconfig.json

```diff
@@ -43,5 +43,7 @@
     "**/*.d.ts",
     "**/*.ts",
     "**/*.svelte",
+    "prettier.config.js",
+    "**/rollup.config.js",
```

Comment:
Can we instead use `rollup.config.ts`?

Reply:
Hmm, I'm not sure. The rollup config we have is currently js not ts, and I don't know whether rollup includes a tsc preprocessing+compilation step for ts files. I think we can investigate in follow up work.